src/Controllers/SchemaController.js

```diff
@@ -621,11 +630,26 @@ class SchemaController {
         found = true;
       }
     }
-    if (!found) {
-      // TODO: Verify correct error code
-      throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND,
+     
+    if (found) {
+      return Promise.resolve();
```

Comment:
This seems to have CLP and Pointer Permissions be "pass one" when it should be "pass both"